README.md

```diff
@@ -869,6 +872,36 @@ The trigger binding utilizes SQL [change tracking](https://docs.microsoft.com/sq
 
     > **NOTE:** The leases table contains all columns corresponding to the primary key from the user table and three additional columns named `_az_func_ChangeVersion`, `_az_func_AttemptCount` and `_az_func_LeaseExpirationTime`. If any of the primary key columns happen to have the same name, that will result in an error message listing any conflicts. In this case, the listed primary key columns must be renamed for the trigger to work.
 
+#### Internal State Tables
+
+The trigger functionality creates a number of tables to use for tracking the current state of the trigger. This allows state to be persisted across sessions and for multiple instances of a trigger binding to execute in parallel (for scaling purposes).
+
+In addition, a schema named `az_func` will be created that the tables will belong to.
+
+The login the trigger is configured to use must be given permissions to create these tables and schema. If not then an error will be thrown and the trigger will fail to run.
+
+If the tables are deleted or modified, then unexpected behavior may occur. To reset the state of the triggers first stop all currently running functions with trigger bindings and then either truncate or delete the tables - the next time a function with a trigger binding is started it will recreate the tables as necessary.
```

Comment:
Put commas in:
- `If not, then an error will be thrown` or use 'Otherwise'.
- `To reset the state of the triggers, first stop all currently running functions`
- `the next time a function with a trigger binding is started, it will recreate the tables as necessary.`

Reply:
Done (sorta - I basically copied everything into word and just took the suggestions there to add commas where it suggested)